src/main/java/com/ryanharter/autoparcel/AutoParcelExtension.java

```diff
@@ -51,16 +48,20 @@ public boolean mustBeAtEnd(Context context) {
   }
 
   @Override
-  public String generateClass(final Context context, final String className, final String classToExtend, boolean isFinal) {
+  public String generateClass(final Context context, final String className,
+                              final String classToExtend, boolean isFinal) {
     return getParcelableCode(context, className, classToExtend, context.properties());
   }
 
   String getParcelableCode(Context context, String className, String classToExtend,
-                                   Map<String, ExecutableElement> properties) {
+                           Map<String, ExecutableElement> properties) {
+    validateProperties(context.processingEnvironment(), properties);
```

Comment:
My thought process here might not be valid once the final property consumption is in place, but at this point *all* properties get sent to applicable, while only non-consumed properties get send to the actual code generation.  So it's possible that some pseudo-properties (like `int describeContents()`) will have been consumed by other extensions and shouldn't be included in the Parcelable implementation.